tekton-task-images/conftest/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2
```

Comment:
I would suggest using `8.3` to get a [patched version of OpenSSL](https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?tag=8.2&container-tabs=security&gti-tabs=unauthenticated)